README.md

```diff
@@ -193,6 +193,20 @@ This codebase follows [single-direction margin declarations](https://csswizardry
 
 Margins are generally preferred to padding for whitespace (due to [margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing)), but the latter can be used when relevant.
 
+Accessibility
+------------
+
+### Screen Reader Observations
+
+#### Voiceover (Mac)**
+
+Download Link Icon
```

Comment:
We're talking general approaches to take, so I don't think we need to mention the specific pattern in this case.